keps/sig-storage/20190204-kubernetes-csi-release-process.md

```diff
@@ -242,6 +259,10 @@ driver.
 ## Graduation Criteria
 
 - Test results are visible in GitHub PRs and test failures block merging.
+- Test results are visible in the [SIG-Storage
+  testgrid](https://k8s-testgrid.appspot.com/sig-storage-kubernetes).
```

Comment:
We may want a new sub-dashboard for csi testing to distinguish it from in-tree tests.

Reply:
I added that as alternative solution. Let's get something working and then decide what's more useful.